### PR TITLE
Hello world Backend

### PR DESCRIPTION
**Pull request by looh16**
Create an API endpoint that return a random greeting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,55 +10,18 @@ After you're finished please remove all the comments and instructions!
 
 </div>
 
-<!-- TABLE OF CONTENTS -->
+# 📖 [hello-rails-back-end] <a name="about-project"></a>
```
Comment (Lucas-Masaba) [inline README.md:13]: - [ ] In this README, please add a link to your React Front-End. This is a project requirement.

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @looh16  👋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️ 

- [ ] Please add a link to your React front-end project in your README to proceed

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by descholar-ceo — APPROVED** [review]
Hi @looh16 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Implemented all remequirements ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by looh16** [comment]
Thanks!

**Pull request merged**